zrml/swaps/src/tests.rs

```diff
@@ -1434,10 +1453,64 @@ fn swap_exact_amount_in_exchanges_correct_values_with_cpmm(
     });
 }
 
+#[test]
+fn swap_exact_amount_in_exchanges_correct_values_with_cpmm_with_fees() {
+    ExtBuilder::default().build().execute_with(|| {
+        let asset_bound = Some(_1 / 2);
+        let max_price = Some(_2);
+        // ALICE swaps in BASE / 0.9; this results in adjusted_in ≈ BASE in
+        // `math::calc_out_given_in` so we can use the same numbers as in the test above!
+        let asset_amount_in = 11_111_111_111;
+        let asset_amount_out = 9_900_990_100;
+        frame_system::Pallet::<Runtime>::set_block_number(1);
+        ASSETS.iter().cloned().for_each(|asset| {
+            let _ = Currencies::deposit(asset, &ALICE, _25);
+            let _ = Currencies::deposit(asset, &BOB, _10000);
+        });
+        assert_ok!(Swaps::create_pool(
+            BOB,
+            ASSETS.iter().cloned().collect(),
+            ASSETS.last().unwrap().clone(),
+            0,
+            ScoringRule::CPMM,
+            Some(BASE / 10),
```

Comment:
Since the other numbers in the test are not based on `MaxSwapFee`, changing `MaxSwapFee` will break the test. I'd rather not use the constant for that reason.